DOC-562-Redis-Support-Changes (#509)
Macrometacorp/docs: docs/queryworkers/redis/index.md

Response from Redis server:
`(integer) 37`

### SINTERSTORE

> SINTERSTORE gives wrong value when existing key is given as destination.

Reviewer: ```suggestion
SINTERSTORE gives the wrong value when an existing key is given as the destination.

```

We will add three different keys with SADD:

```js
[
  "SADD",
  "key_1",
  "a",
  "b",
  "c",
  "d"
]
```

```js
[
  "SADD",
  "key_2",
  "c"
]
```

```js
[
  "SADD",
  "key_3",
  "a",
  "c",
  "e"
]
```

> With SINTERSTORE we will try to store set in existing key:

Reviewer: ```suggestion
With SINTERSTORE, we will try to store the set in an existing key:

```

```js
[
  "SINTERSTORE",
  "key_2",
  "key_1",
  "key_2",
  "key_3"
]
```

Result will be:

```js
{
  "code": 200,
  "result": 0
}
```

Response from Redis server:
`(integer) 1`

:::note

> For Redis SET and SORTED SET datatype commands, using same keys for the command will give a syntax error.

Reviewer: ```suggestion
For Redis SET and SORTED SET datatype commands, using the same keys for the command gives a syntax error.

```

:::

:::note

> For Redis MSET and HMSET commands, using same keys for the command will give a invalid command args error.

Reviewer: ```suggestion
For Redis MSET and HMSET commands, using same keys for the command will give an invalid command args error.

```

Response from Redis server:
`(integer) 37`

### SINTERSTORE

SINTERSTORE gives the wrong value when an existing key is given as the destination.

We will add three different keys with SADD:

```js
[
  "SADD",
  "key_1",
  "a",
  "b",
  "c",
  "d"
]
```

```js
[
  "SADD",
  "key_2",
  "c"
]
```

```js
[
  "SADD",
  "key_3",
  "a",
  "c",
  "e"
]
```
With SINTERSTORE, we will try to store the set in an existing key:

```js
[
  "SINTERSTORE",
  "key_2",
  "key_1",
  "key_2",
  "key_3"
]
```

Result will be:

```js
{
  "code": 200,
  "result": 0
}
```

Response from Redis server:
`(integer) 1`

:::note
For Redis SET and SORTED SET datatype commands, using the same keys for the command gives a syntax error.

> 

Reviewer 2: ```suggestion

For Redis MSET and HMSET commands, using same keys for the command will give an invalid command args error.
```

Merge these note boxes together and remove the second one.

:::

> :::note

Reviewer 2: ```suggestion
```

> For Redis MSET and HMSET commands, using same keys for the command will give an invalid command args error.

Reviewer 2: ```suggestion
```

> :::

Reviewer 2: ```suggestion
```
